.github/workflows/ci.yml

```diff
@@ -16,6 +16,41 @@ jobs:
         uses: actions/setup-node@v2
         with:
           node-version: ${{ steps.get-versions.outputs.node }}
+          cache: yarn
+      - run: yarn install --frozen-lockfile --ignore-scripts
```

Comment:
so we're installing the same thing multiple times in parallel and also rebuilding multiple times directly and indirectly? I don't think that's what we want…

Reply:
installing is mitigated by using the yarn cache, most of the time it's a low cost and it's not like it's an expense for us.
since this runs in parallel, the separate jobs will be completed faster than serially.

what do you mean with rebuilding? for the lint job no build is done, for the test job only the `prepare` script is run which builds the Mux elements. not the examples